torchvision/models/feature_extraction.py

```diff
@@ -172,8 +175,19 @@ def _warn_graph_differences(train_tracer: NodePathTracer, eval_tracer: NodePathT
     warnings.warn(msg + suggestion_msg)
 
 
+def _get_leaf_modules_for_ops():
+    members = inspect.getmembers(torchvision.ops)
+    result = []
+    for _, obj in members:
+        if inspect.isclass(obj) and issubclass(obj, torch.nn.Module):
+            result.append(obj)
+    return tuple(result)
```

Comment:
Non blocking: Due to the variable size of the result, it would be more natural to declare this a list. The mutability won't be an issue as you don't use it as default arg any more.